src/components/Cards/cardsStyles.tsx

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@mui/styles';
+
+export const cardsStyles = makeStyles(() => ({
+  cont: {
```

Comment:
why only "cont"? Are you paying for each letter in naming? make it more readable and understandable